neurons/calculator/calculator.py

```diff
@@ -0,0 +1,81 @@
+# encoding=utf8  
+import math
+
+from core.NeuronModule import NeuronModule, InvalidParameterException
+from core import Utils
+
+
+class Calculator(NeuronModule):
+    def __init__(self, **kwargs):
+
+        super(Calculator, self).__init__(**kwargs)
+
+        """
+        Ideally, a parser, lexer, tokenizer would be the best option for a
+        calculator app. But this works ok for simple operations...
+        """
+
+        # get operation from brain
+        self.operation  = kwargs.get('operation', None)
+
+        # 3 following lines force STT to math operators
+        self.operation  = self.operation.replace('divisé par', '/')
```

Comment:
Yes, it makes sense to isolate Neuron logic from language issues (especially before the project gets too big). As mentioned in other comment, I don't know if you've considered a course of action for internationalization but it might interesting to have locale/language specific helper classes to resolve STT/TTS issues. In can refactor this, it was mainly to increase the success rate of Kalliope's divisions ;)